src/ports/postgres/modules/convex/test/keras_model_arch_table.sql_in

```diff
@@ -0,0 +1,124 @@
+    FROM pg_attribute WHERE attrelid = 'test_keras_model_arch_table'::regclass
+        AND attname = 'model_id';
+SELECT assert(UPPER(atttypid::regtype::TEXT) = 'JSON', 'model_arch column should be JSON type' ) FROM pg_attribute WHERE attrelid = 'test_keras_model_arch_table'::regclass
+        AND attname = 'model_arch';
+SELECT assert(UPPER(atttypid::regtype::TEXT) =
+    'DOUBLE PRECISION[]', 'model_weights column should be DOUBLE PRECISION[] type')
+    FROM pg_attribute WHERE attrelid = 'test_keras_model_arch_table'::regclass
+        AND attname = 'model_weights';
+
+/*  model id should be 1 */
+SELECT assert(model_id = 1, 'Wrong model_id written by keras_create_model')
+    FROM test_keras_model_arch_table;
+
+/* model arch should be valid json, with all fields accessible with json operators */
+SELECT assert((model_arch->>'a') = '1', 'Cannot parse model_arch json in model table.')
+    FROM test_keras_model_arch_table;
+SELECT assert((model_arch->>'b') = '2', 'Cannot parse model_arch json in model table.')
+    FROM test_keras_model_arch_table;
+SELECT assert((model_arch->'c')->>0 = '4', 'Cannot parse model_arch json in model table.')
+    FROM test_keras_model_arch_table;
+SELECT assert((model_arch->'c')->>1 = '5', 'Cannot parse model_arch json in model table.')
+    FROM test_keras_model_arch_table;
+SELECT assert((model_arch->'c')->>2 = '6', 'Cannot parse model_arch json in model table.')
+    FROM test_keras_model_arch_table;
+/* model_weights should be set to null, since this is not a warm start */
+SELECT assert(model_weights IS NULL, 'model_weights should be NULL after keras_create_model() called.') FROM test_keras_model_arch_table;
+
+
+/* Test model creation where valid table exists */
+SELECT keras_create_model('test_keras_model_arch_table', '{"config" : [1,2,3]}');
+SELECT keras_create_model('test_keras_model_arch_table', '{"config" : [8,4,0]}');
+SELECT assert(model_arch->'config'->>0 = '1', 'Cannot parse model_arch json in model table.')
+    FROM test_keras_model_arch_table WHERE model_id = 2;
+SELECT assert(model_arch->'config'->>1 = '2', 'Cannot parse model_arch json in model table.')
+    FROM test_keras_model_arch_table WHERE model_id = 2;
+SELECT assert(model_arch->'config'->>2 = '3', 'Cannot parse model_arch json in model table.')
+    FROM test_keras_model_arch_table WHERE model_id = 2;
+SELECT assert(model_arch->'config'->>0 = '8', 'Cannot parse model_arch json in model table.')
+    FROM test_keras_model_arch_table WHERE model_id = 3;
+SELECT assert(model_arch->'config'->>1 = '4', 'Cannot parse model_arch json in model table.')
+    FROM test_keras_model_arch_table WHERE model_id = 3;
+SELECT assert(model_arch->'config'->>2 = '0', 'Cannot parse model_arch json in model table.')
+    FROM test_keras_model_arch_table WHERE model_id = 3;
+
+/* Test deletion where valid table exists */
+SELECT keras_delete_model('test_keras_model_arch_table', 2);
```

Comment:
The name of the function created to do this is `delete_keras_model`, but the tests call `keras_delete_model`. This would fail. This is the case in other similar test cases too I guess.